src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -112,6 +112,19 @@ class ReviewCollapsibleChapter extends React.Component {
     return chapterTitle;
   };
 
+  getPageTitle = rawPageTitle => {
```

Comment:
This allows us to use functions for page titles (functions already work for chapter titles).